docs/src/main/asciidoc/config-reference.adoc

```diff
@@ -674,19 +711,51 @@ extensions. Therefore, the `quarkus.` prefix should **never** be used for applic
 
 === Build Time configuration
 
-Some Quarkus configurations only take effect during build time, meaning it is not possible to change them at runtime. These
-configurations are still available at runtime but as read-only and have no effect in Quarkus behaviour. A change to any
-of these configurations requires a rebuild of the application itself to reflect changes of such properties.
+Some Quarkus configurations only take effect during build time (compile), meaning it is not possible to change it at
+runtime. These configurations are still available at runtime but as read-only and have no effect in Quarkus
```

Comment:
Is it true if they are build time only i.e. not build time and runtime fixed?

Reply:
If you are using a config root to access the configuration, yes, but you can always query the configuration name manually, since at the config source level, we don't distinguish by phase.

I didn't want to go into too many details, since I consider this to be mainly an intro guide to the configuration pieces. This is already documented in the config extension part. If you feel that we need to give more details here, we can do it.